presentation/src/main/java/com/knocklock/presentation/navigation/KnockLockNavHost.kt

```diff
@@ -39,6 +39,7 @@ fun NavGraphBuilder.settingGraph(
         composable(route = NavigationRoute.SettingGraph.Setting.route) {
             SettingRoute(
                 modifier = modifier,
+                navigateByUserSettings = { navController.navigate(NavigationRoute.SettingGraph.Password.route) },
```

Comment:
혹시 navigateByUserSettings 람다 내부에서 추후 다른곳으로 이동할 여지가 있을까요??

있다면 매개변수로 Route 이름을 전달받거나 
```suggestion
                navigateByUserSettings = { route ->
                   navController.navigate(route )
                },
```
없다면 네이밍을 수정하면 좋을 것 같아요!
```suggestion
                navigateToPasswordInputScreen= { navController.navigate(NavigationRoute.SettingGraph.Password.route) },
```

Reply:
위에 민욱이의 navigateToPasswordInputScreen 네이밍이 더 명확하게 보입니당

Reply:
반영했습니다!